target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -158,6 +158,13 @@ tplink,cpe210-v3)
 tplink,cpe610-v1)
 	ucidef_set_led_netdev "lan" "LAN" "tp-link:green:lan" "eth0"
 	;;
+tplink,re355-v1|\
+tplink,re450-v1)
+	ucidef_set_led_netdev "lan_data" "LAN Data" "tp-link:green:lan_data" "eth0" "tx rx"
+	ucidef_set_led_netdev "lan_link" "LAN Link" "tp-link:green:lan_link" "eth0" "link"
+	ucidef_set_led_wlan "wlan2g" "WLAN 2.4 GHz" "tp-link:blue:wlan2g" "phy1tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN 5 GHz" "tp-link:blue:wlan5g" "phy0tpt"
```

Comment:
The triggers are already defined in the device tree. I got confused because i thought there have to be LED entries inside /etc/config/system. Removed those two lines and merged it with the `tplink,re450-v2` trigger definition.